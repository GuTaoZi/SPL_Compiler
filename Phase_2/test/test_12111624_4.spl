struct ww {
    int a;
};

struct qq {
    struct ww ss[10086];
} qwe[998244353];

int main(){
    qwe[qwe[0].ss[0].a] = qwe[3];

    qwe[qwe[0].ss[1]] = qwe[1];
}