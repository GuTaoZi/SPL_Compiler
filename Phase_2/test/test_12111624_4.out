Error type 12 at Line 12: index: 'qwe[0].ss[1]' should be an integer
